dev/django.Dockerfile

```diff
@@ -40,8 +40,6 @@ RUN pip install \
 # Use a directory name which will never be an import name, as isort considers this as first-party.
 WORKDIR /opt/django-project
 
-# Setup nginx to proxy localhost:9000 to minio:9000
-COPY ./dev/nginx.conf /etc/nginx/nginx.conf
 COPY ./dev/docker-entrypoint.sh /docker-entrypoint.sh
 RUN chmod +x /docker-entrypoint.sh
 ENTRYPOINT ["/docker-entrypoint.sh"]
```

Comment:
We should also remove this entry point and the bash file it's using.

Reply:
Additionally, we can remove the `nginx` installation on line 21.

Reply:
Changed in [055bbe6](https://github.com/atlascope/atlascope/pull/122/commits/055bbe606f991f7f5ef5eb570ab568d4f3bee47f)